cp3-networkpolicy/egress/license-service-reporter/ibm-license-service-reporter-egress.yaml

```diff
@@ -0,0 +1,11 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ibm-license-service-reporter-deny-all-egress
```

Comment:
why are we adding deny-all for a service? it's what customers do - we should provide only the allow policyes, not deny policies

Reply:
Pentest - CS 4.7.0 - 154899: Unrestricted Internet Access/Outbound Connections shown problems with access from the ibm-license-service-reporter to outside. Base on ZERO trust additional Egress was added even if Kubernetes documentation say that such restriction should be set as default.

Reply:
No - for pentest, we should apply namespace-level egress deny-all. ZERO trust is the cluster admin construct, not application level construct. Please remove this network policy.